design/courses/courses-mockup.json

```diff
@@ -37,7 +59,23 @@
             "gradeLevel": "College",
             "subjectLevel": "Beginner",
             "members": [],
-            "steps": []
+            "steps": [
+                {
+					"stepTitle": "Introductory Steps",
+					"description": "Send us your resume again (along with a short description of yourself and what you would like to be called) to vi+resume@ole.org. Create a GitHub account then watch Open Learning Exchange and Follow Leonard and Dogi on GitHub. In GitHub we Watch organization repositories and Follow individuals. Look for the Watch button at the top right of the page when you are in a repository. Also follow each other to see what others in the group are doing.",
+					"attachment": ""
+				},

```

Comment:
indentation issue

Reply:
@lmmrssa i check in my vscode and sublime text editor the indentation looks ok.

Reply:
it was using Tab instead of space